packages/components/src/components/DistributionChart.tsx

```diff
@@ -28,6 +28,7 @@ export type DistributionPlottingSettings = {
   logX: boolean;
   /** Set the y scale to be exponential by deault */
   expY: boolean;
+  truncateToNthci: number;
```

Comment:
Could this be either a number or false/null? Not sure if that would work smoothly in Typescript, but it would help here. In most cases we don't want this.